code-challenges-03/array-binary-search.js

```diff
@@ -0,0 +1,20 @@
+
+module.exports = exports = {};
+
+exports.binarySearch = (arr, key) => {
+  let start = 0;
+  let end = arr.length-1;
+  let mid = Math.ceil(arr.length / 2);
+
+  while(start < end){
+    mid = Math.ceil((start + end) / 2);
+    if(arr[mid] < key){
+      start = mid;
+    } else if(arr[mid] > key){
+      end = mid;
+    }else{
+      return mid;
+    }
+  }
+};
```

Comment:
😫 this extra `},` is breaking your export. Hopefully your linter points out that this curly brace is causing problems